docs-new/docs/graphql/core/api-reference/explain.mdx

```diff
@@ -173,5 +173,5 @@ Microsoft SQL Server backend:
 
 ## Disabling Explain API
 
-The Explain API is part of the `Metadata API <metadata_apis>` and can
+The Explain API is part of the [Metadata API](/graphql/core/api-reference/metadata-api/index.mdx#metadata-apis) and can
```

Comment:
`#metadata-apis` hash paramis not necessary as this links to the main doc itself.